src/Microsoft.DocAsCode.Plugins/IHrefInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Plugins
+{
+    public interface IHrefInfo
+    {
+        string FileInSource { get; }
+        string FileInDest { get; }
+        string TargetFileInSource { get; }
```

Comment:
>TargetFileInSource [](start = 15, length = 18)

the naming confuses me and I am unable to figure out what they mean from names....